3 - PB/Documentazione esterna/Piano di Qualifica/cruscotto/cruscotto.json

```diff
@@ -0,0 +1,876 @@
+	   "PB":"Come anche per il periodo RTB, i documenti hanno rilevato un maggior numero di errori ortografici nel periodo di maggiore stesura. Nonostante ciò, la maggiore confidenza con la redazione dei documenti, ha permesso di ridurre notevolmente il numero di errori riscontrati. In particolare: \n - *Documentazione esterna*: \n\t - #json_pdp_v: il documento ha subito modifiche consistenti per rispondere alle criticità riscontrate alla valutazione RTB, aumentando il grado di analisi e dettaglio. Nonostante ciò, il numero di errori ortografici presenti non è mai stato eccessivo, per poi stabilizzarsi dallo Sprint 18; \n\t - #json_pdq_v: il documento ha subito modifiche consistenti per rispondere alle criticità riscontrate alla valutazione RTB, aumentando il grado di analisi e dettaglio. In particolare è stata redatta l'intera sezione di metriche di qualità riferite al prodotto e la sezione dei test condotti. In particolare infatti, durante i periodi 18, 19 e 20 il documento ha visto la redazione delle sezioni dedicate alle suite di test; \n\t - #json_adr_v: il documento ha subito modifiche consistenti per rispondere alle criticità riscontrate alla valutazione RTB, aumentando il grado di analisi e dettaglio. L'aggiornamento di tale documento infatti ha avuto la priorità per permettere al gruppo di iniziare il processo di codifica in modo solido. La versione aggiornata del documento è stata inoltre valutata positivamente anche dal Professor Cardin. \n\t - #json_glo_v: Il glossario non ha subito grandi modifiche, stabilizzando il numero di errori ortografici a 0, e le modifiche successive, di dimensioni ridotte, non hanno presentato errori. \n\t - #json_st_v: Il documento ha visto la sua redazione dal periodo 17 al 20, precedente all'inizio del processo di codifica, in modo da evidenziare e fissare le scelte progettuali (design). Il numero di errori ortografici è stato contenuto; \n\t - #json_man_v: Il documento non ha riscontrato un numero elevato di errori ortografici. La sua redazione è iniziata nello Sprint 19, in quanto si voleva attendere che il prodotto software assumesse una forma delineata in modo concreto. \n - *Documentazione interna*: \n\t - #json_ndp_v: il documento non ha subito modifiche in termini di struttura e contenuto, quanto più un'estensione per affrontare le nuove attività che il gruppo ha dovuto svolgere, in particolare in merito ai processi di progettazione, codifica e verifica. La sua redazione è conclusa nello Sprint 20, stabilizzando a 0 il numero di errori individuati.",
```

Comment:
In particolare infatti => Infatti ("in particolare" è stato già scritto precedentemente)